src/manager.rs

```diff
@@ -228,37 +229,43 @@ impl Manager {
 
     pub fn enable_poll_with_intensity(&mut self, value: &ValueID, intensity: u8) -> bool {
         unsafe {
-            extern_manager::manager_enable_poll_with_intensity(self.ptr, value.ptr, intensity)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
+            extern_manager::manager_enable_poll_with_intensity(self.ptr, &cvid, intensity)
         }
     }
 
     pub fn enable_poll(&mut self, value: &ValueID) -> bool {
         unsafe {
-            extern_manager::manager_enable_poll(self.ptr, value.ptr)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
+            extern_manager::manager_enable_poll(self.ptr, &cvid)
         }
     }
 
     pub fn disable_poll(&mut self, value: &ValueID) -> bool {
         unsafe {
-            extern_manager::manager_disable_poll(self.ptr, value.ptr)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
+            extern_manager::manager_disable_poll(self.ptr, &cvid)
         }
     }
 
     pub fn is_polled(&mut self, value: &ValueID) -> bool {
         unsafe {
-            extern_manager::manager_is_polled(self.ptr, value.ptr)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
+            extern_manager::manager_is_polled(self.ptr, &cvid)
         }
     }
 
     pub fn set_poll_intensity(&mut self, value: &ValueID, intensity: u8) {
         unsafe {
-            extern_manager::manager_set_poll_intensity(self.ptr, value.ptr, intensity)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
+            extern_manager::manager_set_poll_intensity(self.ptr, &cvid, intensity)
         }
     }
 
     pub fn get_poll_intensity(&mut self, value: &ValueID) -> u8 {
         unsafe {
-            extern_manager::manager_get_poll_intensity(self.ptr, value.ptr)
+            let cvid = extern_value_id::value_id_from_packed_id(value.home_id, value.id);
+            extern_manager::manager_get_poll_intensity(self.ptr, &cvid)
         }
     }
 }
```

Comment:
just wondering why you use `value_id_from_packed_id` here whereas for other APIs you created ValueID from the cpp wrapper ? The latter seems less verbose and more readable.